octopus_deploy/assets/logs/octopus_deploy.yaml

```
@@ -0,0 +1,51 @@
+id: octopus_deploy
+metric_id: octopus-deploy
+backend_only: false
+facets: null
```

This is not technically an issue, but an empty list looks cleaner to read
```suggestion
facets: []
```